src/transforms/stack.js

```diff
@@ -81,7 +81,11 @@ function stack(x, y = one, kx, ky, {offset, order, reverse}, options) {
   const [Y2, setY2] = column(y);
   Y1.hint = Y2.hint = lengthy;
   offset = maybeOffset(offset);
-  order = maybeOrder(order, offset, ky); // TODO shorthand -order with reverse?
+  if (typeof order === "string" && order.startsWith("-")) {
+    order = order.slice(1);
+    reverse = !reverse;
+  }
```

Comment:
Same here re. reverse. And also as commented above, *order* can be a field name, and I don’t think we should support *-field* syntax since it is more likely to conflict with a user-supplied column name.